### DISCUSSION TITLE
Why doesn't get errors useFormContext or useForm

### DISCUSSION THREAD

**Discussion by Phibee** [react-hook-form/react-hook-form#6351] [Q&A] [answered]
Hi guys, i just want to ask why is it not getting yup validation errors  inside RHF FormProvider? It doesn't show error message.  I'm using React (Typescript) even the errors is not available on Typescript

**Comment by bluebill1049** [comment, marked as answer]
`formState: { errors }`

**Reply by Phibee** [reply]
Yes that's correct. but values is undefined.

**Reply by Phibee** [reply]
it's okay now sir. thanks @bluebill1049.

**Reply by bluebill1049** [reply]
weclome.
